tests/FakeItEasy.IntegrationTests/AssertionsTests.cs

```diff
@@ -101,52 +92,26 @@ public void Should_fail_when_calls_to_same_method_on_different_instances_did_not
         {
             // Arrange
             Exception exception;
+            var fooOne = A.Fake<IFoo>();
+            var fooTwo = A.Fake<IFoo>();
 
-            using (var scope = Fake.CreateScope())
-            {
-                var fooOne = A.Fake<IFoo>();
-                var fooTwo = A.Fake<IFoo>();
-
-                // Act
-                fooOne.Bar();
-                fooTwo.Bar();
+            // Act
+            fooOne.Bar();
+            fooTwo.Bar();
 
-                exception = Record.Exception(() =>
+            exception = Record.Exception(() =>
+            {
+                using (var context = A.CallOrderingContext())
                 {
-                    using (scope.OrderedAssertions())
-                    {
-                        A.CallTo(() => fooTwo.Bar()).MustHaveHappened();
-                        A.CallTo(() => fooOne.Bar()).MustHaveHappened();
-                    }
-                });
-            }
+                    A.CallTo(() => fooTwo.Bar()).MustHaveHappened().InOrder(context);
+                    A.CallTo(() => fooOne.Bar()).MustHaveHappened().InOrder(context);
+                }
+            });
 
             // Assert
             exception.Should().BeAnExceptionOfType<ExpectationException>();
         }
 
-        [Test]
-        public void Should_throw_when_starting_new_ordered_assertions_scope_when_one_is_already_opened()
-        {
-            // Arrange
-            Exception exception;
-
-            using (Enumerable.Empty<ICompletedFakeObjectCall>().OrderedAssertions())
-            {
-                // Act
-                exception = Record.Exception(
-                    () =>
-                    {
-                        using (Enumerable.Empty<ICompletedFakeObjectCall>().OrderedAssertions())
-                        {
-                        }
-                    });
-            }
-
-            // Assert
-            exception.Should().BeAnExceptionOfType<InvalidOperationException>();
-        }

```

Comment:
Oh! I didn't know that this was a thing. I'm glad we don't have to worry about supporting nested ordering scopes.